tests/test_io_dummy_param.rs

```diff
@@ -46,7 +46,7 @@ mod test {
             public_circuit: public_circuit.clone(),
             d: 3,
             encoding_sigma: 0.0,
-            hardcoded_key_sigma: 3.0,
+            hardcoded_key_sigma: 0.0,
```

Comment:
shouldn't we keep this 3?

Reply:
We should use 0 for all error parameters in the dummy params test to ensure that intermediate values are computed as expected.